cmd/extract.go

```diff
@@ -117,11 +112,12 @@ in preparation to build the final docker image.`,
 			cmdArgs = append([]string{"create"}, cmdArgs...)
 			cmdArgs = append(cmdArgs, stackImage)
 			err = execAndWaitReturnErr(cmdName, cmdArgs, Debug)
-
 			if err != nil {
+
 				Error.log("docker create command failed: ", err)
 				dockerRemove(extractContainerName)
-				os.Exit(1)
+				return err
```

Comment:
Why is the pattern different here? I see we aren't wrapping `err` in an `errors.Errorf`... and we left the `Error.log()`... is there a good reason?